terraform/main.tf

```
@@ -308,6 +320,94 @@ resource "aws_iam_role_policy" "api_task-publish_to_arpa_audit_report_queue" {
   policy      = data.aws_iam_policy_document.publish_to_arpa_audit_report_queue.json
 }
 
+module "arpa_treasury_report" {
+  source                   = "./modules/sqs_consumer_task"
+  namespace                = "${var.namespace}-arpa_treasury_report"
```

I'll see if I can devise a better way to avoid these problems entirely in the future, but I have a feeling this is going to hit a name-length issue on some resource... this should work:
```suggestion
  namespace                = "${var.namespace}-treasury_report"
```